tox.ini

```diff
@@ -0,0 +1,22 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py36, pypy3, pylama
+
+[testenv]
+commands =
+  coverage erase
+  coverage run --source=django_consul_memcached setup.py test
+  coverage report
+deps =
+     coverage
+     -rrequirements.txt
+     -rtests-requirements.txt
+
+[testenv:pylama]
+commands = pylama django-consul-memcached
```

Comment:
The directory is `django_consul_memcached` now.

Reply:
Done